Misc/vcs_info-examples

```diff
@@ -160,7 +160,7 @@ zstyle ':vcs_info:git*+set-message:*' hooks git-untracked
 
 +vi-git-untracked(){
     if [[ $(git rev-parse --is-inside-work-tree 2> /dev/null) == 'true' ]] && \
-        git status --porcelain | grep '??' &> /dev/null ; then
+        git status --porcelain 2> /dev/null | grep -q '^?? ' ; then
```

Comment:
Why is git's stderr discarded?

Shouldn't the question marks be escaped? (preëxisting issue)

Reply:
I discarded git's stderr because it shouldn't pollute the prompt in case of failure.

Question marks are treated literally by grep's default [BRE syntax](https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap09.html#tag_09_03), so this is fine.